File: 11-01-broadcast-receiver/java-11-01-broadcast-receiver/app/src/main/java/com/example/broadcast/FirstReceiver.java
package com.example.broadcast;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.widget.Toast;


public class FirstReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        if (intent.getAction().equals("com.example.broadcast.FIRST_MESSAGE")) {
            Toast.makeText(context, "리시버가 수행되었습니다", Toast.LENGTH_SHORT).show();
        }
    }
}

File: 11-01-broadcast-receiver/kotlin-11-01-broadcast-receiver/app/src/main/java/com/example/broadcast/FirstReceiver.kt
package com.example.broadcast

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.widget.Toast

class FirstReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        if (intent.action == "com.example.broadcast.FIRST_MESSAGE") {
            Toast.makeText(context, "리시버가 수행되었습니다", Toast.LENGTH_SHORT).show()
        }
    }
}
